### PR TITLE
lots of changes and fixes cleanup too

### PR DESCRIPTION
**Pull request by DevX32**
## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

**Comment by DevX32** [comment]
need some changes i will do it soon

**Comment by DevX32** [comment]
need more changes wait for it

**Review by Manason — CHANGES_REQUESTED** [review]
Before working further on the implementation of this PR, we should discuss the merits of the idea of an admin car.

File: server/main.lua [hunk 1]
```diff
@@ -122,6 +122,31 @@ RegisterNetEvent('qb-admin:server:changeplayerdata', function(Selected, Selected
     PlayerDataOptions[Selected](Target, Input)
 end)
 
+RegisterNetEvent('qb-admin:server:SaveCar', function(mods, vehicle, plate)
```
[the remark this answers is not in the document]
Comment (DevX32) [reply server/main.lua:125]: i will do all thing what u said

File: server/commands.lua [hunk 2]
```diff
@@ -12,6 +12,13 @@ lib.addCommand('noclip', {
     TriggerClientEvent('qb-admin:client:noclip', source)
 end)
 
+lib.addCommand('admincar', {
+    help = 'Toggle Admin Car',
+    restricted = 'admin',
+}, function(source)
+    TriggerClientEvent('qb-admin:client:SaveCar', source)
```
[the remark this answers is not in the document]
Comment (DevX32) [reply server/commands.lua:19]: its saves the car / give owership of veh/car

**Pull request closed without merging**